src/DocBlock/Annotation.php

```diff
@@ -187,6 +168,33 @@ public function getTag()
         return $this->tag;
     }
 
+    /**
+     * @return TypeExpression
+     *
+     * @internal until 2.19
+     */
+    public function getTypeExpression()
+    {
+        return new TypeExpression($this->getTypesContent(), $this->namespace, $this->namespaceUses);
+    }
+
+    /**
+     * @return null|string
+     *
+     * @internal until 2.19
```

Comment:
```suggestion
     * @internal
```

to speed up review. let's separately discuss if we would like to make it applied for BC promise